docs/diagrams/LogicClassDiagram.puml

```diff
@@ -28,17 +29,20 @@ HiddenOutside ..> Logic
 
 LogicManager .right.|> Logic
 LogicManager -right->"1" AddressBookParser
-AddressBookParser ..> XYZCommand : creates >
+AddressBookParser ..> NormalCommand : creates >
+AddressBookParser ..> UndoableCommand : creates >
```

Comment:
`AddressBookParser` needs to point towards some `ABCCommand` or `XYZCommand` extending from `UndoableCommand` instead, as `UndoableCommand` is `abstract`.